### DISCUSSION TITLE
How to start streampark locally

### DISCUSSION THREAD

**Discussion by Kick156** [apache/streampark#2390] [Q&A] [answered]
I want to debug my modified code on the local idea,

But I don't see any content about local operation and debugging on the official website,

Who else has relevant information for me to refer to.

**Comment by NightRunner** [comment, marked as answer]
Recently, the secondary development page has been deleted, but you can reference  :[development](https://github.com/apache/incubator-streampark-website/blob/d680d982e28716eb5ed62a21027f74b787f9dc32/i18n/zh-CN/docusaurus-plugin-content-docs/current/user-guide/3-development.md)

**Reply by Kick156** [reply]
Thank you. I'll try it later。
